CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Added: More starting locations have been added for all areas.
 - Added: Progressive pickups can now be configured to be placed vanilla.
+- Added: New cosmetic to shuffle music either by song type or full shuffle.
```

Comment:
I'm not sure if this should have the word `[New cosmetic] option` added, so I just leave it as a comment.

Reply:
I'll fix it. Does this pr also need the persisted options updated?

Reply:
yes